presto-tests/src/test/java/io/prestosql/execution/EventsAwaitingQueries.java

```diff
@@ -64,7 +67,7 @@ MaterializedResult runQueryAndWaitForEvents(@Language("SQL") String sql, int num
         }
 
         eventsCollector.waitForEvents(10);
-
+        Thread.sleep(extraWaitTime.toMillis());
```

Comment:
Add a comment why we're doing this, otherwise the first impression is "flaky!" and the second is "let's fix that"